releases/release-1.32/release-notes/maps/pr-125577-map.yaml

```diff
@@ -0,0 +1,30 @@
+pr: 125577
+releasenote:
+  text: Added a /statusz endpoint for kube-apiserver endpoint
```

Comment:
```suggestion
  text: Added a `/statusz` endpoint for kube-apiserver endpoint
```

Reply:
```suggestion
  text: Added a `/statusz` endpoint for kube-apiserver endpoint.
```

Reply:
Can you try running `krel release-notes validate` to check for the yaml formatting and punctuation?